class.jetpack-admin.php

```diff
@@ -51,6 +52,39 @@ private function __construct() {
 
 		// Add module bulk actions handler
 		add_action( 'jetpack_unrecognized_action', array( $this, 'handle_unrecognized_action' ) );
+
+		// If the site has Jetpack Anti-Spam, change the Akismet menu label accordingly.
+		$site_products = Jetpack_Site_Products::get();
+		if ( in_array( 'jetpack_anti_spam', array_column( $site_products, 'product_slug' ), true ) ) {
```

Comment:
This only detects Jetpack Anti-Spam yearly, we have to detect both periods, monthly and yearly.

```suggestion
		$anti_spam_products = [ 'jetpack_anti_spam_monthly', 'jetpack_anti_spam' ];
		
		if ( ! empty( array_intersect( $anti_spam_products, array_column( $site_products, 'product_slug' ) ) ) ) {
```

Reply:
Great catch Clemente! I did what you suggested but I had to change the array syntax from `[...]` to `array(...)` because of the linter.